### PR TITLE
Update process docs

### LINKED ISSUE #576: Document pull requests usage
- [x] What metadata fields should be assigned and by whom
  - [ ] Create github-actions issues for things that can be automated
- [x] What to do when PR is ready for re-review (click re-request review)
- [x] When to use draft PRs ~(practically never?)~

### PR DESCRIPTION
**Pull request by AlekSi**
Closes #576.

## Readiness checklist

* [ ] I added tests for new functionality or bugfixes.
* [ ] I ran `task all`, and it passed.
* [ ] I added/updated comments for both exported and unexported top-level declarations (functions, types, etc).
* [ ] I checked comments rendering with `task godocs`.
* [x] (for maintainers only) I set Reviewers, Assignee, Labels, Project and project's Sprint fields.
* [x] I marked all done items in this checklist.

### CODE REVIEW

File: .github/PROCESS.md [hunk 1]
```diff
@@ -4,8 +4,6 @@ This file contains some process notes for FerretDB Inc. engineers and long-term
 If you are a community contributor, feel free to ignore all this!
 We will do all that for you so you can focus on what is really interesting for you.
 
-We encourage diversity between us as different points of view expand our vision and contribute to the project's success.
```
Comment (AlekSi) [inline .github/PROCESS.md:7]: That's still true, but it was out-of-place there. We still have it in our team contract and Code of Conduct

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -20,5 +20,6 @@ This PR closes #{issue_number}.
 * [ ] I ran `task all`, and it passed.
 * [ ] I added/updated comments for both exported and unexported top-level declarations (functions, types, etc).
 * [ ] I checked comments rendering with `task godocs`.
-* [ ] (for maintainers only) I set Reviewers, Assignee, Labels, Project and project's Sprint fields.
+* [ ] (for maintainers only) I set Reviewers ([`@FerretDB/core`](https://github.com/orgs/FerretDB/teams/core)),
```
Comment (rumyantseva) [inline .github/PULL_REQUEST_TEMPLATE.md:23]: Fun fact is that you didn't do it for the current PR :D

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -20,5 +20,6 @@ This PR closes #{issue_number}.
...
+      Assignee, Labels, Project and project's Sprint fields.
 * [ ] I marked all done items in this checklist.
```
Comment (rumyantseva) [inline .github/PULL_REQUEST_TEMPLATE.md:25]: You didn't do it for the current PR! :D

**Review by rumyantseva — APPROVED** [review]
LGTM

**Review by w84thesun — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 3 above)
Comment (AlekSi) [reply .github/PULL_REQUEST_TEMPLATE.md:25]: Sorry, WDYM?

Comment (rumyantseva) [reply .github/PULL_REQUEST_TEMPLATE.md:25]: Already fixed. When I was checking the PR, this item hadn't been checked for it.

**Pull request merged**